dropwizard-jdbi3/src/main/java/io/dropwizard/jdbi3/jersey/LoggingJDBIExceptionMapper.java

```diff
@@ -0,0 +1,35 @@
+package io.dropwizard.jdbi3.jersey;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.dropwizard.jersey.errors.LoggingExceptionMapper;
+import org.jdbi.v3.core.JdbiException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.ext.Provider;
+import java.sql.SQLException;
+
+/**
+ * Iterates through a {@link JdbiException}'s cause if it's a {@link SQLException} otherwise log as normal.
+ */
+@Provider
+public class LoggingJdbiExceptionMapper extends LoggingExceptionMapper<JdbiException> {
```

Comment:
I guess the file also should be named `LoggingJdbiExceptionMapper` instead `LoggingJDBIExceptionMapper`

Reply:
Yeah the joys of file paths on Windows with Git... I'll rename those properly.